config/puma.rb

```diff
@@ -1,10 +1,15 @@
-#!/usr/bin/env puma
+# frozen_string_literal: true
 
-if ENV['RAILS_ENV'] !~ /development|test/
-  directory '/rails_app'
-end
+app_path = File.expand_path(File.dirname(File.dirname(__FILE__)))
+dev_env = 'development'
+rails_env = ENV['RAILS_ENV'] || dev_env
```

Comment:
Is there a convention in Ruby for when you would use `ENV.fetch` vs this?

Reply:
yes, when you want a default. I'll update
```suggestion
rails_env = ENV.fetch('RAILS_ENV', dev_env)
```